src/dask_awkward/layers/layers.py

```diff
@@ -5,16 +5,15 @@
 from typing import TYPE_CHECKING, Any, Literal, Protocol, TypeVar, Union, cast
 
 import dask
-
-_dask_uses_tasks = hasattr(dask, "_task_spec")
-
 from dask.blockwise import Blockwise, BlockwiseDepDict, blockwise_token
 from dask.highlevelgraph import MaterializedLayer
 from dask.layers import DataFrameTreeReduction
 from typing_extensions import TypeAlias
 
 from dask_awkward.utils import LazyInputsDict
 
+_dask_uses_tasks = hasattr(dask, "_task_spec") and hasattr(dask.blockwise, "Task")
```

Comment:
The second check here is probably enough? It's the change in blockwise specifically that broke things.